trainers/gnn_base.py

```diff
@@ -208,5 +232,11 @@ def train(self, train_data_loader, n_epochs, valid_data_loader=None):
             self.save_summary(summary)
             if self.output_dir is not None and self.rank == 0:
                 self.write_checkpoint(checkpoint_id=epoch)
+            
+            if self.summaries['valid_loss'].min() < best_loss:
+                best_loss = self.summaries['valid_loss'].min()
+                patience = 0
```

Comment:
Were you trying to implement early stopping here? The patience never gets set to anything besides 0